packages/flutter/lib/src/animation/animations.dart

```diff
@@ -625,3 +632,25 @@ class AnimationMean extends CompoundAnimation<double> {
   @override
   double get value => (first.value + next.value) / 2.0;
 }
+
+/// An animation that tracks the maximum of two other animations.
+///
+/// The [value] of this animation is the maximum of the values of 
+/// [first] and [next].
+class AnimationMax<T extends num> extends CompoundAnimation<T> {
+  AnimationMax(Animation<T> first, Animation<T> next): super(first: first, next: next);
```

Comment:
This turned up in our benchmarks as an undocumented public member (our baseline is zero).  Can you please submit a PR to fix?